### DISCUSSION TITLE
which case to print out?

### DISCUSSION THREAD

**Discussion by ljmerza** [CircuitSetup/Expandable-6-Channel-ESP32-Energy-Meter#153] [Q&A]
i have a single 6 channel board and want to print out the case but its not clear which parts i need to print out 

https://github.com/CircuitSetup/Expandable-6-Channel-ESP32-Energy-Meter/blob/master/Hardware/README.md

**Comment by wjcarpenter** [comment]
Through trial and error, I have discovered a couple of possibilities:

You can print `6channel_case.stl` and `6channel_case_lid.stl`. This is a fairly simple 2-piece snap-together case. When I printed it, the "tab" on the lid were slightly too big to fit the "slots" on the base (probably due to the nature of 3D printing). I'm trying to resolve that with minor scaling, but you can also just carefully trim the lid "tabs". There is also `6channel_meter_case.stl`, which, as far as I can tell, is the same two pieces in a single STL.

Or, you can print the 3-piece case consisting of `6channel_energy_meter_2piece_case_main_board_large_half.stl`, `6channel_energy_meter_2piece_case_main_board_small_half.stl`, and `6channel_energy_meter_2piece_case_cover.stl`. The small and large halves fit together but probably need some glue to hold them. The lid snaps in place.

**Comment by wjcarpenter** [comment]
I have had mixed results in printing these cases. That's probably mostly because I'm using a low-end Ender 3v2 printer. So far, I've been printing at 0.2mm layer height with a 0.4mm nozzle. I've printed them standing up and laying down, with supports where required. 

If anyone has suggestions for improving prints for these particular cases, I'd love to hear them.
